app/src/main/java/br/com/connectattoo/ui/confirmation/ConfirmationFragment.kt

```diff
@@ -36,9 +36,9 @@ class ConfirmationFragment : BaseFragment<FragmentConfirmationBinding>() {
         repository = AuthRepository()
 
         viewLifecycleOwner.lifecycleScope.launch {
-            val token = DataStoreManager.getStringToken(requireContext(), API_TOKEN)
+            val token = arguments?.getString("token")
             try {
-                val result = repository.verifyUserConfirmation("Bearer $token")
+                val result = repository.verifyUserConfirmation(token ?: "")
```

Comment:
Nesse caso, quando for feita uma chamada pra API com o token tendo o valor de "" (string vazia) pode ser que seja retornado um erro. Sendo assim, não seria apropriado fazer uma verificação pra evitar que a chamada seja feita? Ou é certo que o token nunca será nulo?

Reply:
Foi adicionado mais um check de null para que não aja chamada na api desnecessária.